frontend/src/App.js

```diff
@@ -13,13 +13,6 @@ import {loadUser} from "./actions/auth";
 import BlogItem from "./components/blog/BlogItem";
 
 class App extends Component {
-
-    state = {
-        blog_list: [],
```

Comment:
Where is this functionality moved ? and as per last review you were required to move these numeric values to constants?